src/firebase/events-delete.js

```diff
@@ -0,0 +1,6 @@
+const initializeHttpClient = require('./http/initialize-http-client')
+
+module.exports = eventPaths =>
+  initializeHttpClient().then(firebaseHttpClient =>
+    eventPaths.map(eventPath => firebaseHttpClient.delete(eventPath))
+  )
```

Comment:
Soooo... hopefully creating 100+ promises attached to 100+ HTTP requests doesn't use up all the memory in the Netlify/Lambda function. I tried running them sequentially, but it took more than the allocated 10 seconds.

Since I'm taking a mostly-functional approach and only fetching event keys instead of whole events, garbage collection should be able to keep up during the first run of 100+ events. After that, there should only be ~50 events to work with at any given time. 😎